server/controllers/ticket/create.php

```diff
@@ -114,22 +122,43 @@ public function handler() {
                 $this->sendMailStaff($staff->email);
             }
         }
+        
+        Log::createLog('CREATE_TICKET', $this->ticketNumber);
 
         Response::respondSuccess([
             'ticketNumber' => $this->ticketNumber
         ]);
+           
+    }
 
-        if(!Controller::isUserSystemEnabled() && !Controller::isStaffLogged()) {
-            $session = Session::getInstance();
-            $session->createTicketSession($this->ticketNumber);
-        }
-        
-        Log::createLog('CREATE_TICKET', $this->ticketNumber);
+    private function isEmailInvalid(){
+        return  (Session::getInstance()->sessionExists() && User::getUser(Session::getInstance()->getUserId(),'id')  &&  $this->email && !(User::getUser(Session::getInstance()->getUserId(),'id')->email == $this->email));
```

Comment:
this is too long please put `Session::getInstance()` in a variable and `Session::getInstance()->getUserId()` too